nautobot/dcim/api/serializers.py

```diff
@@ -524,6 +525,7 @@ class Meta:
 class DeviceSerializer(TaggedModelSerializerMixin, NautobotModelSerializer):
     face = ChoiceField(choices=DeviceFaceChoices, allow_blank=True, required=False)
     config_context = serializers.SerializerMethodField()
+    cluster = serializers.SerializerMethodField()
```

Comment:
I think the preferred approach here would be to implement this as a `NautobotHyperlinkedRelatedField`, similar to `VLANSerializer.location` and `PrefixSerializer.location`.

Reply:
I followed the pattern from `VLANSerializer.location`. For my own understanding, this doesn't provide full backwards compatibility, since the `cluster` attribute is no longer returned in API results, correct? As implemented, `NautobotHyperlinkedRelatedField` only lets us write to the `cluster` attribute